docs/modules/operation/pages/alarms/alarm-advanced-search.adoc

```diff
@@ -0,0 +1,53 @@
+
+[[ga-advanced-alarm-search]]
+= Advanced Alarm Search
+
+Often users wish to query the alarm list with more customizations to narrow down the search.
+The Horizon alarm list page has a button to filter your searches based on different criteria.
+
+The following are the criteria which can be searched upon:
+
+* Alarm Text Contains
+** Performs a substring filter on both the log message and the description
+** To negate this search, prefix the inputted text with `-`.
+This will return results which do not contain the provided input text.
+
+* TCP/IP Address Like
+** Performs an `IPLIKE` on the provided IP Address to filter the results.
+** To negate this search, prefix the inputted text with `-`.
+This will return results which does not satisfy the `IPLIKE` function.
+
+* Node Label Contains
+** Performs a substring filter on the Node Label.
+** To negate this search, prefix the inputted text with `-`.
+This will return results which does not contain the provided input text.
+
+* Severity
+** Filter the alarms list based on the alarm severity.
+** To negate this search, toggle the checkbox below titled "Negate Severity".
+This will return results which does not match the set Severity.
+
+* Service
+** Filter the alarms list based on the alarm service.
+** To negate this search, toggle the checkbox below titled "Negate Service".
+This will return results which does not match the set Service.
+
+* Alarm type
+** Select this dropdown to include Only Alarms, Only Situations or Both.
+
+* Category
+** Filter the alarms list based on the nodes category.
+** To negate this search, toggle the checkbox titled "Negate Category".
+This will return results which does not match the set Category.
+
+* Filter for Alarm's First Event After
+** Select the timestamp to show alarms after the time of the first event relating to this alarm.
+
+* Filter for Alarm's First Event Before
+** Select the timestamp to show the alarms before the time of the first event relating to this alarm.
+
+* Filter for Alarm's Last Event After
+** Select the timestamp to show the alarms after the time of the last event relating to this alarm.
+
+* Filter for Alarm Last Event Before
+** Select the timestamp to show the alarms before the time of the last event relating to this alarm.
```

Comment:
```suggestion
** Select the timestamp to show the alarms before the time of the alarm's last event.
```
Wouldn't this provide the same results as "Filter for Alarm's First Event After"?
Can one combine these before/after alarm filters? If so, does the logic prevent them from cancelling each other out?

Reply:
An Alarm has 2 dates:
```
firsteventtime        | 2022-06-13 11:23:28.719+00
lasteventtime         | 2022-06-29 09:33:48.387+00
```
These 4 filters restrict the list based on these 2 fields. I thinking there is valid reasoning for them not to cancel out. You could have a long list of alarms where you only want a subset based maybe on the outer limits, or inner limits or both.

Granted, there does not look to be validation done for the case where  firsteventtime > lasteventtime, which i guess should never be a valid search